planner/core/testdata/plan_suite_out.json

```diff
@@ -879,7 +879,7 @@
     "Cases": [
       {
         "SQL": "select a from t where c is not null",
-        "Best": "IndexReader(Index(t.c_d_e)[[-inf,+inf]])->Projection"
+        "Best": "IndexReader(Index(t.f)[[NULL,+inf]])"
```

Comment:
This is because `c is not null` is constantly true, so it is removed; while index(f) is smaller than index(c,d,e) regarding row size, so the index chosen is changed.